.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+<!--
+Thanks for opening an issue! A few things to keep in mind:
+
+- The issue tracker is only for bugs and feature requests.
+- If you need general help please post to StackOverflow http://stackoverflow.com/questions/tagged/activeadmin
+- Before reporting a bug, please search all issues, closed and open, and try reproducing your issue against
+  the latest version.
+-->
+
+### Expected behavior
+
+<!-- What do you think should happen? -->
+
+### Actual behavior
+
+<!-- What actually happens? -->
+
+### How to reproduce
+
+<!-- 
+
+Your best chance of getting this bug looked at quickly is to provide a **code sample** or an **executable test case** demonstrating the expected behavior that is not occurring.
+
+See CONTRIBUTING for more details.
```

Comment:
@varyonic You're absolutely right! This is indeed viewable in the preview screen but I'm not sure either of the usefulness. And yes, the pop up contributing link seems enough!